### PR TITLE
Mobile first

### PR DESCRIPTION
**Pull request by chigozie1803**
In this pull request, i:
Added a html file
Added a css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>chigozie ezeilo portfolio</title>
+</head>
+<body>
+
+    <div class="wrapper">
+        <header>
+            <nav>
+                 <img src="./image/Logo Placeholder.svg" alt="" class="logo">
```
Comment (Santiago220991) [inline index.html:15]: Kindly wrap this line of code inside an `a` tag. This is for three reasons: convention, semantics, and user experience(UX). :nerd_face:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                 <button class="menu"><i class="fafa-bars"></i></button>
+             </nav>
+        </header>
+        <section class="header-container">
+            <h1>Hey There.</h1>
+            <h2>I'm Chigozie</h2>
```
Comment (Santiago220991) [inline index.html]: Kindly switch the tags between these two lines of code. "I'm Chigozie"  should be wrapped in an `h1` tag this way google (and other search engines) knows about a specific topic in this case about you. :nerd_face: :nerd_face:

File: style.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+h1,
+h2 {
+  width: 327px;
+  height: 124px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  display: flex;
```
Comment (Santiago220991) [inline style.css:63]: - Please make sure that your project sticks as much as possible to the Figma template provided. Kindly reduce the space between the lines of the title. :+1: :smiley:  :smiley: 
- Also make sure please to use the font provided by the Figma file. :eyes: 
image

File: style.css [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+li {
+  display: flex;
+  justify-content: space-between;
+}
+
+nav.logo {
+  margin-bottom: 2px;
+  margin-left: 12px;
+}
```
Comment (Santiago220991) [inline style.css:41]: Kindly make sure that your using the same font size provided in the Figma file. :eyes: 
image

File: style.css [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+.wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
```
Comment (Santiago220991) [inline style.css:11]: - [OPTIONAL] Kindly make your mobile version responsive until the desktop screen size. :+1: :+1: :smiley: 

image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @chigozie1803 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
:heavy_check_mark: No linter errors.
:heavy_check_mark: Follower GitHub flow.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- I think is a good idea to make your README file more professional. You could add more details about how to use your project. It is also a good idea to fill the author's info with your information and links to your social media pages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (Nyame-Wolf) [reply style.css:41]: [OPTIONAL] Kuddos on making the changes highlighted by the previous reviewer, however, On makking the change, the alignment of this navbar got messed up. Please put this two in a flex container and they will be automatically aligned.

Screenshot 2022-09-24 at 20 11 42

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @chigozie1803  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**